code/modules/vehicle/ambulance.dm

```diff
@@ -124,3 +124,11 @@
 		else
 			amb.bed = src
 			to_chat(usr, "You hook the bed to the ambulance.")
+
+/obj/vehicle/ambulance/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/borg/upgrade/vtec))
+		vehicle_move_delay = 1
+		qdel(I)
+		to_chat(user,"<span class='notice'>You upgrade [src] with [I].</span>")
+		return
+	return ..()
```

Comment:
I feel like this should be innate on the `obj/vehicle` level

Reply:
You want every vehicle to be upgradable with vtec? 
If so, then what should be done with the already fast vehicles like the speedbike which has 0 move delay?

Reply:
it should early return if the move delay is less than or equal to 1

Reply:
Done! This is now also a buff for the snowmobile too hehe.
Tested and works fine!